pkg/descheduler/evictions/evictions.go

```diff
@@ -111,10 +118,15 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node,
 		reason += " (" + strings.Join(reasons, ", ") + ")"
 	}
 	if pe.maxPodsToEvictPerNode != nil && pe.nodepodCount[node]+1 > *pe.maxPodsToEvictPerNode {
-		metrics.PodsEvicted.With(map[string]string{"result": "maximum number reached", "strategy": strategy, "namespace": pod.Namespace}).Inc()
+		metrics.PodsEvicted.With(map[string]string{"result": "maximum number of pods per node reached", "strategy": strategy, "node": pod.Spec.NodeName}).Inc()
```

Comment:
Please keep the original `"namespace": pod.Namespace`. The metrics are reported for pods. The particular node/namespace from which a pod was not allowed to be evicted can be read from a specific pod. Also, we are not interested in every pod's case, just an overall picture. With the `namespace` label we can see which namespaces are getting rejected the most.